src/observe.ts

```diff
@@ -22,6 +22,7 @@ export const initPerformanceObserver = () => {
     po('resource', initResourceTiming);
   }
   perfObservers[3] = po('layout-shift', initLayoutShift);
+  po('element', initElementTiming);
```

Comment:
Have you tested this in Firefox, Safari, and Edge? I guess this code works only on Chrome right?

Reply:
Good point:

- Firefox does warn
![firefox](https://user-images.githubusercontent.com/3313870/83352241-17c24c80-a342-11ea-8f55-3d170fb7ef4c.jpg)


- legacy Edge doesn't seem to care
![legacy edge](https://user-images.githubusercontent.com/3313870/83352223-fcefd800-a341-11ea-9273-f0b812d7926b.jpg)


- Chrome and new Edge work as expected:
![Chrome](https://user-images.githubusercontent.com/3313870/83352283-493b1800-a342-11ea-853b-2116dbb20218.png)

![new edge](https://user-images.githubusercontent.com/3313870/83352307-65d75000-a342-11ea-84ce-41e3f801326e.png)


I'll test Safari tomorrow

Reply:
When testing Safari I was getting consistent TypeError even on the live site but it seems that was by design in the V1 spec: https://github.com/w3c/performance-timeline/issues/87

iOS 13.3
![iOS13](https://user-images.githubusercontent.com/3313870/83428776-f1c4a700-a42a-11ea-949a-8cad9850a635.png)

iOS 12.4
![iOS12](https://user-images.githubusercontent.com/3313870/83428979-423c0480-a42b-11ea-8670-a130597c88c3.png)


Safari 13
![safari 13](https://user-images.githubusercontent.com/3313870/83428954-34867f00-a42b-11ea-9914-98dd7786aae0.png)

Safari 12
![safari 12](https://user-images.githubusercontent.com/3313870/83428930-29cbea00-a42b-11ea-8d55-245e0a979d09.png)

Live Safari 12
![live safari 12](https://user-images.githubusercontent.com/3313870/83428908-233d7280-a42b-11ea-9914-8799bbf647b4.png)

Reply:
Interesting, I wonder if we should safeguard this for Safari.

Reply:
The best I could find was https://github.com/GoogleChrome/web-vitals/blob/master/src/lib/observe.ts#L34 which seems quite reasonable

Reply:
Because this is an experimental API, we should probably introduce a configuration on/off options as we do for Resource Timing.

```js
const perfume = new Perfume({
  elementTiming: true,
  analyticsTracker: ({ metricName, data }) => {
    myAnalyticsTool.track(metricName, data);
  })
});
```

Btw side note, we might want to remove the diff view in https://github.com/Zizzamia/perfume.js#element-timing, and just keep what the final version should look like.

Reply:
Done in https://github.com/Zizzamia/perfume.js/pull/127